plugin/pkg/shoot/validator/admission.go

```diff
@@ -357,6 +368,21 @@ func (c *validationContext) validateScheduling(a admission.Attributes, seedListe
 	return nil
 }
 
+func getNumberOfShootsOnSeed(shootLister corelisters.ShootLister, seedName string) (int64, error) {
+	allShoots, err := shootLister.Shoots(metav1.NamespaceAll).List(labels.Everything())
+	if err != nil {
+		return 0, fmt.Errorf("could not list all shoots: %w", err)
+	}
+
+	count := int64(0)
```

Comment:
Out of curiosity: Why `int64`? :)

Reply:
Reasonable scaling expectations I assume ;)

Reply:
Because we want to compare it to an `int64` later on (returned by `allocatableShoots.Value()`), so we need an `int64` here as well :)